packages/solo/package.json

```diff
@@ -26,6 +26,7 @@
     "@agoric/assert": "^0.6.0",
     "@agoric/cache": "^0.3.2",
     "@agoric/cosmic-swingset": "^0.41.3",
+    "@agoric/inter-protocol": "^0.16.1",
```

Comment:
Can/should this be just a `devDependency`?

Reply:
Removed per ^^